src/client/canvas/svg/marker.tsx

```diff
@@ -0,0 +1,20 @@
+import { observer } from 'mobx-react'
+import * as React from 'react'
+
+import { Transform } from '../../math/transform'
+import { MarkerGeometry } from '../geometry/marker_geometry'
+import { Shape } from '../object/shape'
+
+import { svgAppearanceAttributes } from './svg'
+
+type Props = { model: Shape<MarkerGeometry>, world: Transform }
+export const Marker = observer(({ model: {
+  geometry: { x, y, radius, heading },
+  appearance,
+},                                world }: Props) => {
+  const rotation = 135.0 + (180.0 / Math.PI) * Math.atan2(heading.y, heading.x)
+  return <path d='M-1 -1L0 -1A1 1 270 1 1 -1 0Z'
```

Comment:
I propose using a formatting like this (which I use elsewhere), simply because it makes the indentation look visually very similar to regular JavaScript:

```ts
return (
  <path
    d='M-1 -1L0 -1A1 1 270 1 1 -1 0Z'
    transform={`scale(${radius}) rotate(${rotation}) translate(${x}, ${y})`}
    {...svgAppearanceAttributes(appearance)}
  />
)
```

vs an imagined JavaScript version:

```ts
return {
  path: {
    d: '...',
    transform: '...',
    {...}
  }
};
```